docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -0,0 +1,369 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting information about SOFA objects\n",
+    "\n",
+    "An formal description of each convention is given in the [sofar documentation](https://sofar.readthedocs.io/en/stable/resources/conventions.html). You might have noted from the documentation that three different kinds of data types can be stored in SOFA files:\n",
+    "\n",
+    "* **Attributes:**\n",
+    "    Attributes are meta data stored as strings. There are two kinds of attributes. Global attributes give information about the entire data stored in a SOFA file. All entires starting with *GLOBAL* are such attributes. Specific attributes hold meta data for a certain variable. These attributes thus start with the name of the variable followed by an underscore, e.g., *ListenerPosition_Units*. An exception to this rule are the data variables, e.g, *Data_IR* is not an attribute but a double variable.\n",
+    "* **Double Variables:**\n",
+    "    Variables of type *double* store numeric data and can be entered as numbers, lists, or numpy arrays.\n",
+    "* **String Variables:**\n",
+    "    Variables of type *string* store strings and can be entered as strings, lists of string, or numpy string arrays.\n",
+    "\n",
+    "The data can be mandatory, optional, and read only and must have a shape (dimension in SOFA language) according to the underlying convention. Read on for more information.\n",
+    "\n",
+    "To get a quick insight into SOFA objects\n",
+    "\n",
+    "* ``sofa.inspect`` prints the data stored in a SOFA object or at least gives\n",
+    "  the shape in case of large arrays that would clutter the output. This is\n",
+    "  helpful when reading data from an existing SOFA object.\n",
+    "* ``sofa.list_dimensions`` prints the dimensions of the data inside the SOFA\n",
+    "  object.\n",
+    "* ``sofa.get_dimension`` returns the size of a specific dimension.\n",
+    "\n",
+    "For the *SimpleFreeFieldHRIR* SOFA object we have the following dimensions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa.list_dimensions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case, `M` denotes the number of source positions for which HRIRs are available, `R` is the number of ears - which is two - and `N` gives the lengths of the HRIRs in samples. `S` is zero, because the convention does not have any string variables. `C` is always three, because coordinates are either given by x, y, and z values or by their azimuth, elevation and radius in degree."
```

Comment:
```suggestion
    "In this case, `M` denotes the number of source positions for which HRIRs are available, `R` is the number of ears - which is two - and `N` gives the lengths of the HRIRs in samples. `S` is zero, because the convention does not have any string variables. `C` is always three, because coordinates are either given by x, y, and z values or by their azimuth, elevation and radius in degree as defined in the [sofa conventions](https://www.sofaconventions.org/mediawiki/index.php/SOFA_specifications#Coordinate_system)."
```

Reply:
good idea to link this